<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>branch</class>
 <widget class="QDialog" name="branch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>228</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manager</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="city_LineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_LineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="manager_LineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
